MeetU/MeetU/Scripts/Controllers/Meetup/create.js

```diff
@@ -2,7 +2,45 @@
     "use strict";
     angular
         .module('meetupModule')
-        .controller('MeetupCreateController', MeetupCreateController);
+        .controller('MeetupCreateController', MeetupCreateController)
+        .directive('myValidation', function () {
```

Comment:
you should not put a directive here. should be /Scripts/Directives/validation.js

also, myValidation is incorrect, you can use validation or meetupValidation